backend/app/services/device_monitor.py

```
@@ -0,0 +1,224 @@
+        logger.warning("[DeviceMonitor] Monitor not running")
+        return
+
+    _monitor_running = False
+
+    if _monitor_task:
+        _monitor_task.cancel()
+        try:
+            # Don't await in sync context
+            pass
+        except Exception:
+            pass
```

Ruff (0.14.14)

`try`-`except`-`pass` detected, consider logging the exception

(S110)